src/main/java/games/strategy/engine/framework/startup/ui/editors/EmailSenderEditor.java

```diff
@@ -167,8 +168,9 @@ private void testEmail() {
         message = "Email sent, it should arrive shortly, otherwise check your spam folder";
         messageType = JOptionPane.INFORMATION_MESSAGE;
       } catch (final IOException ioe) {
-        message = "Unable to send email; see TripleA Console for details.";
-        ClientLogger.logError(ioe);
+        message = "Unable to send email, check SMTP server credentials: " +
```

Comment:
Checkstyle: "'+' should be on a new line."

Reply:
Fixed in: 8eadf5e